catalog/core/catalog-core-standardframework/src/main/resources/OSGI-INF/blueprint/blueprint.xml

```diff
@@ -370,49 +372,77 @@
         </reference-listener>
     </reference-list>
 
-    <!-- create the ddf bean -->
-    <bean id="ddf" class="ddf.catalog.impl.CatalogFrameworkImpl">
+    <!--
+        Framework properties bean
+    -->
+    <bean id="frameworkProperties" class="ddf.catalog.impl.FrameworkProperties">
```

Comment:
My gut feel is that if you do try to break this thing apart, we'll just end up with the same properties in multiple objects as opposed to everything in this one object. It would also make it annoying down the road if you ever needed to add something to all of the operations.